pyvista/plotting/widgets.py

```diff
@@ -1609,7 +1609,7 @@ def _the_callback(widget, event_id):
         spline_widget.PlaceWidget(bounds)
         spline_widget.SetResolution(resolution)
         if initial_points is not None:
-            spline_widget.InitializeHandles(pyvista.vtk_points((initial_points)))
+            spline_widget.InitializeHandles(pyvista.vtk_points(initial_points))
```

Comment:
Perhaps we should add a test since we're at it, if thete's one we can easily extend.

Reply:
Could this be added in the pre-commit routine?

Reply:
I meant for this line, because codecov said it is uncovered. But good point.

Reply:
Ah, wasn't sure if that's what you meant or not. Both would be good to add

Reply:
Do you mean to add a test for with a tuple arg, i.e. `vtk_points((initial_points,))`?

As for the separate thing about [pyupgrade as a pre-commit hook](https://github.com/asottile/pyupgrade#as-a-pre-commit-hook), is that welcome for this PR?

Reply:
IMO, yes - paring the introduction of that hook with the changes that hook would make/evaluate is most appropriate.

> Do you mean to add a test for with a tuple arg, i.e. vtk_points((initial_points,))?

Not quite. Codecov is showing that the current tests for `add_spline_widget()` leave `initial_points` as `None`. We should expand `test_widget_spline` (in `tests/plotting/test_widgets.py`) to pass an appropriate value for that parameter to get coverage on that line

Reply:
Pre-commit hook is now added. @mwtoews how do you feel about adding the additional test case to make the diff coverage 100% here? Not critical for CI, just "would be nice to have" :)

Reply:
@adeak how does 4e06e195e7ec29d0d87292f04168bb200ddb2a0b look?

Reply:
Works beautifully, thanks!